Resources/Help/analyses/regressionlinearbayesian.md

```diff
@@ -77,8 +77,8 @@ Compares each model against the model selected.
   - Models: Predictors contained in the model.
   - P(M): Prior model probabilities.
   - P(M | data): Posterior probabilites of the models considered.
-  - BFM: Posterior model odds.
-  - BF10 (or BF01): Bayes factor.
+  - BFM: The updating factor by which the prior model odds change into the posterior model odds (e.g., when BFM = 2, the posterior model odds are twice as large as the prior model odds).
```

Comment:
Following `ancovabayesian_nl.md` I'd translate this as:

```
BFM: De update-factor waarmee de prior model odds veranderen in de posterior model odds (bv. wanneer BFM = 2, dan is de posterior model odds twee keer zo groot als de prior model odds).
```

for the other one I'd say:
```
Bayes-factor die een model vergelijkt met ofwel het nulmodel, ofwel het model met de hoogste posterior model kans.
```
where "posterior model kans" is again inspired by `ancovabayesian_nl.md`.

Reply:
Thanks!